app/database/seeds/01_clients.js

```diff
@@ -0,0 +1,33 @@
+exports.seed = function(knex) {
+  // Deletes ALL existing entries
+  return knex('clients').del()
+    .then(function () {
+      // Inserts seed entries
+      return knex('clients').insert([
+        {
+          id: 1,
+          first_name: 'Elias',
+          last_name: 'Fang',
+          email: 'efang@ucsd.edu',
+          is_recipient: true,
+          job_id: 6
```

Comment:
Are there jobs seeds as well for the job_id to map to?

Reply:
I'm not sure if I'm understanding your question but the job_id field should correspond to the id field in the 'jobs' table.

Reply:
Yeah are those jobs being added to the table?

Reply:
Yes